### ISSUE TITLE
[Health] Build failing on iOS, Flutter version 1.17

### ISSUE THREAD

**Issue by DiogoTheCoder** [cph-cachet/flutter-plugins#87]
**Plugin name & version**
Health 1.0.2

**Describe the bug**
When running `flutter run`, getting error message:
```bash
Xcode's output:
↳
    === BUILD TARGET firebase_core OF PROJECT Pods WITH CONFIGURATION Debug ===
    /Users/diogocosta/.pub-cache/hosted/pub.dartlang.org/health-1.0.2/ios/Classes/SwiftHealthPlugin.swift:107:25: warning: comparing non-optional value
    of type '[HKQuantitySample]' to 'nil' always returns true
                if (samples != nil){
                    ~~~~~~~ ^  ~~~
```

**To Reproduce**
Upgrade to Flutter version 1.17
Add to dependency, `health: ^1.0.2`

**Screenshots**
![image](https://user-images.githubusercontent.com/31896387/81722123-db08e100-9478-11ea-9e40-ba0c5b007872.png)

**Flutter doctor**
```bash
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.17.0, on Mac OS X 10.15.3 19D76, locale en-GB)

[!] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor --android-licenses
[✓] Xcode - develop for iOS and macOS (Xcode 11.4.1)
[✓] Android Studio (version 3.6)
[!] IntelliJ IDEA Ultimate Edition (version 2019.3.1)
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
[✓] VS Code (version 1.44.2)
[✓] Connected device (1 available)

! Doctor found issues in 2 categories.
```

**Smartphone(s) / Emulator**
```
iPhone 11 Pro Max • DD5795DD-0FD0-4C01-932F-B8C23656272C • ios • com.apple.CoreSimulator.SimRuntime.iOS-13-4 (simulator)
```

**Comment by DiogoTheCoder** [comment]
Fixed issue with build, had to remove ```ios/Runner.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings``` after upgrading to v1.17. https://github.com/flutter/flutter/issues/56567

However, this warning should still be resolved.

**Comment by thomasnilsson** [comment]
Should be fixed in version 1.0.6 :)
